### DISCUSSION TITLE
The use of using the word "Center" mark as a filter for articles as read, filters everything

### DISCUSSION THREAD

**Discussion by beerisgood** [FreshRSS/FreshRSS#6399] [Q&A] [answered]
### Describe the bug

Similar bug as https://github.com/FreshRSS/FreshRSS/issues/6386 but with word 'Center'.
'\Center' is okay.

### To Reproduce

Go to 'feed'
Click on 'manage feed'
Scroll down to 'filter actions'
add 'Center' to "mark article as read" part
scroll a little bit down to archive and clear all articles
scroll to bottom and reload articles
all articles from that feed are marked as read

### Expected behavior

Only articles with specific filter words should be marked as read

### FreshRSS version

1.23.1

### Environment information

PHP version: [8.3.6]
Installation type: [manual via zip archive]
Uberspace v7

### Additional context

used installation guide: https://lab.uberspace.de/guide_freshrss/

**Comment by Alkarex** [comment]
What feed have you used to test?
My first guess is that the test is done on a feed that has `Center` in its HTML.

**Comment by beerisgood** [comment]
The site is: https://www.macrumors.com/
and the feed URL is: http://feeds.macrumors.com/MacRumors-All

**Comment by Alkarex** [comment, marked as answer]
The feed entries contain:

```html
<div class="center-wrap">...</div>
```

which becomes, after sanitizing:

```html
<div data-sanitized-class="center-wrap">...</div>
```

and we search in body, we search in all text, including HTML, hence the match on "center".

To search only in title, you can try `intitle:Center`. Or if a word is expected before, search with a space such as `" Center"`.

We may get some regex search in the future. (Edit: https://github.com/FreshRSS/FreshRSS/pull/6706 )

**Reply by beerisgood** [reply]
Thanks!

I only wonder why does the filter not only check the title that FreshRSS displays by default?

**Reply by Alkarex** [reply]
By default, the search is in title + body

**Reply by beerisgood** [reply]
Can this be changed somehow to title only?

**Reply by Alkarex** [reply]
> To search only in title, you can try `intitle:Center`

**Reply by beerisgood** [reply]
I meant generally as standard, not specifically via filter.
